### ISSUE TITLE
integrating new tutorials into the master branch

### ISSUE THREAD

**Issue by edublancas** [ploomber/soopervisor#118]
the [tutorials](https://github.com/ploomber/soopervisor/tree/tutorials/kind/doc) branch contains revamped tutorials. More information on why we created these new tutorials is in: https://github.com/ploomber/soopervisor/issues/109 - essentially, the current ones do not contain the output of each command, which makes them hard to follow

a few things:

- tutorials require some extra files to work. currently, the tutorials assume the files exist locally, but we should replace them with `curl` commands. see this: https://ploomber.io/blog/ds-platform-part-i/ for an example of what we're looking for
- ignore the [argo-pykernel.ipynb](https://github.com/ploomber/soopervisor/blob/tutorials/kind/doc/argo-pykernel.ipynb) file, since we'll be using the argo.ipynb version (the latter uses the bash kernel)
- the bash kernel gets a little crazy when the commands output a lot of text, so you might wanna redirect the output to a file
- the tutorials should replace the old ones (e.g., use the `argo.ipynb` and delete the existing one

@RodolfoFerro

**Comment by idomic** [comment]
@edublancas can we close this?
If stuff are still missing please list them
